src/beam.js

```diff
@@ -652,8 +652,31 @@ export class Beam extends Element {
     }
   }
 
+  // return upper level beam direction. L for Left, R for Right, B for Both
+  lookupBeamDirection(duration, prev_tick, tick, next_tick) {
+    if (duration === '4') {
+      return 'L';
+    }
+
+    const lookup_duration =  `${Flow.durationToNumber(duration) / 2}`;
+    const prev_note_gets_beam = prev_tick < Flow.durationToTicks(lookup_duration);
+    const next_note_gets_beam = next_tick < Flow.durationToTicks(lookup_duration);
+    const note_gets_beam = tick < Flow.durationToTicks(lookup_duration);
+
+    if (prev_note_gets_beam && next_note_gets_beam && note_gets_beam) {
+      return 'B';
```

Comment:
Can we use constants here instead of strings?

Reply:
Thanks for reviewing! That means is it okay to just declare constants in `lookupBeamDirection` function?
![image](https://user-images.githubusercontent.com/34280965/59725458-aee4e000-9268-11e9-8ade-d86a340f94d1.png)

Reply:
Make them `const` scoped to this file (at the top). So `const BEAM_LEFT = 'L'`. And use them in the caller.

Reply:
Yap

Reply:
At first I declared consts in `lookupBeamDirection` scope but I make `const` scoped to beam file now